draft-ietf-mls-protocol.md

```diff
@@ -1576,18 +1582,35 @@ struct {
           opaque confirmation_tag<0..255>;
     }
 } MLSPlaintextTBS;
+
```

Comment:
Please break this out into its own sub-section titled "Plaintext Authentication" or similar, and an explainer that `membership_tag` is only necessary when sending MLSPlaintext over-the-wire